runtime/src/bank.rs

```diff
@@ -2063,11 +2058,6 @@ impl Bank {
 
         if Stakes::is_stake(account) {
             self.stakes.write().unwrap().store(pubkey, account);
-        } else if storage_utils::is_storage(account) {
```

Comment:
@sakridge this will cause a snapshot to fail?

Reply:
Don't believe so, because these fields are not covered by any hash in the snapshot. As long as it deserializes the same it should be ok.